infra/base-images/msan-builder/compiler_wrapper.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+from __future__ import print_function
+import os
+import sys
+import subprocess
+
+
+def Is32Bit(args):
+  M32_BIT_ARGS = [
+      '-m32',
+      '-mx32',
+  ]
+
+  return any(arg in M32_BIT_ARGS for arg in args)
+
+
+def GetCompilerArgs(args):
+  compiler_args = args[1:]
+
+  if Is32Bit(args):
+    # 32 bit builds not supported.
+    compiler_args.extend([
+        '-fno-sanitize=memory',
+        '-fno-sanitize-memory-track-origins',
```

Comment:
Just to make sure: these arguments are position independent, right? Other compilations flags cannot overwrite them?

Reply:
These are added to the very end, so should override any previous compilation flags.